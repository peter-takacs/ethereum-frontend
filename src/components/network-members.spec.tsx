import * as React from 'react';
import * as ReactDOM from 'react-dom';
import * as renderer from 'react-test-renderer';
import NetworkMembers from './network-members';

test('Renders correctly', () => {
    const component = renderer.create(
        <NetworkMembers members={
            ['0x0123', '0xABCD']
        } />
    )
    expect(component.toJSON()).toMatchSnapshot();
})