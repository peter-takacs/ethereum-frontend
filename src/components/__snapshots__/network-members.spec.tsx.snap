// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Renders correctly 1`] = `
<div>
  <header>
    Educator network members:
  </header>
  <ul>
    <li>
      0x0123
    </li>
    <li>
      0xABCD
    </li>
  </ul>
</div>
`;
